selenium_tests/admin/test_skip_logic.py

```diff
@@ -0,0 +1,94 @@
+import json
+from django.urls import reverse
+from wagtail.core.models import Site
+
+from iogt_users.factories import AdminUserFactory
+from questionnaires.factories import SurveyFactory, SurveyFormFieldFactory
+from selenium_tests.base import BaseSeleniumTests
+from selenium_tests.pages import WagtailAdminPage
+
+
+class AdminPanelSeleniumTests(BaseSeleniumTests):
+    def setUp(self):
+        Site.objects.all().delete()
+        self.setup_blank_site()
+        admin_user = AdminUserFactory()
+        self.visit_login_page().login_user(admin_user)
+        self.survey01 = SurveyFactory(
+            parent=self.home,
+            owner=admin_user,
+            thank_you_text="[{\"type\": \"paragraph\", \"value\": \"<p>Thankyou for completing the survey</p>\"}]"
+            )
+
+    def test_skip_logic_validation_errors_on_admin_panel(self):
+        skip_logic = json.dumps(
+            [
+                {
+                    "type": "skip_logic",
+                    "value": {
+                        "choice": "true",
+                        "skip_logic": "next",
+                        "question": None
+                    }
+                },
+                {
+                    "type": "skip_logic",
+                    "value": {
+                        "choice": "false",
+                        "skip_logic": "next",
+                        "question": None
+                    }
+                }
+            ]
+        )
+        SurveyFormFieldFactory(label='Q1', page=self.survey01, required=True, field_type='checkbox',
+                               skip_logic=skip_logic)
+        SurveyFormFieldFactory(label='Q2', page=self.survey01, required=True, field_type='checkbox',
+                               skip_logic=skip_logic)
+        SurveyFormFieldFactory(label='Q3', page=self.survey01, required=True, field_type='checkbox',
+                               skip_logic=skip_logic)
+
+        self.visit_url(reverse('wagtailadmin_pages:edit', args=(self.survey01.id,)))
+
+        admin_page = WagtailAdminPage(self.selenium)
+        admin_page.select_skip_logic('survey_form_fields-0-skip_logic-0-value-skip_logic')
+        admin_page.select_skip_logic_question("survey_form_fields-0-skip_logic-0-value-question_1", "3")
```

Comment:
This does not help us to easily understand what is happening in this test. There is a better way to do this, but for now, I would like to see something like:

```
admin_page.select_skip_to_question(question=0, answer=0, skip_to=3)
```

I don't want to see any details of the HTML page like CSS selectors and element IDs in the test - these are low-level details that should be hidden at this level.